applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -605,16 +620,26 @@ int le_audio_play_pause(void)
 {
 	int ret;
 
-	if (playing_state) {
-		playing_state = false;
+	if (paused) {
+		paused = false;
+		return 0;
+	}
+
+	if (active_stream.stream == NULL || active_stream.stream->ep == NULL) {
+		LOG_WRN("Stream or endpoint not set");
+		return -EPERM;
+	}
 
+	if (active_stream.stream->ep->status.state == BT_BAP_EP_STATE_STREAMING) {
+		paused = true;
 		ret = bt_bap_broadcast_sink_stop(broadcast_sink);
 		if (ret) {
-			LOG_ERR("Failed to stop broadcast sink: %d", ret);
+			LOG_WRN("Failed to stop broadcast sink: %d", ret);
 			return ret;
 		}
 	} else {
-		playing_state = true;
+		LOG_WRN("Current stream not in streaming state");
+		return 0;
```

Comment:
Probably we can remove this line, since the return 0 in line 645 cannot be removed.

Reply:
Done